### DISCUSSION TITLE
Mancanza del campo countryIso nella classe Client

### DISCUSSION THREAD

**Discussion by marionee** [fattureincloud/api#345] [Q&A] [answered]
Buongiorno

utilizzo la vostra SDK Java per Fatture in Cloud, specificamente la versione 2.0.18. Ho notato un'incongruenza che vorrei portare alla vostra attenzione.

Nella classe Entity, esiste un campo countryIso che permette di specificare il codice ISO del paese. Questa opzione è molto utile e facilita la gestione internazionale delle entità.

Tuttavia, ho notato che tale campo non è presente nella classe Client, che è utilizzata per memorizzare le informazioni sull'anagrafica del cliente. Questa classe è molto simile alla classe Entity, ma manca di alcuni campi, tra cui appunto countryIso.

Potete fornire qualche spiegazione su questa discrepanza? Sarebbe molto utile avere il campo countryIso anche nella classe Client per mantenere la coerenza e facilitare l'integrazione con sistemi esterni.

Grazie.

**Comment by emajo** [comment]
Ciao @marionee effetivamente è un nostro errore, il campo `country_iso` dovrebbe essere presente anche nella classe Client, lo includeremo sicuramente nel prossimo rilascio.

Grazie,
Emanuele.

**Comment by emajo** [comment, marked as answer]
Ciao @marionee, abbiamo rilasciato la documentazione e le relative SDKs contenenti il campo country_iso.

Grazie,
Emanuele.

**Reply by dmigliacci** [reply]
riesumo questa discussione, perchè sto avendo problemi nella impostazione della **country** del customer di una fattura.
io ricevo le country nel formato IT, IE etc ...

ma ricevo l'errore "The data.entity.country field is not valid."

come devo convertirle?

grazie in anticipo

**Reply by marionee** [reply]
@dmigliacci, nel tuo caso va utilizzato il campo countryIso e non country nell'oggetto entity.

**Reply by dmigliacci** [reply]
Eh immaginavo, ma purtroppo l'oggetto Customer, dentro il document, non ha countryIso. Non c'è neanche nell'ultima sdk... 😞

**Reply by marionee** [reply]
Dentro IssuedDocument dovresti avere Entity che ha quel campo.

**Reply by dmigliacci** [reply]
ok scusa non avevo capito, quello giustamente c'è ... lo provo e vi dico, intanto grazie

**Reply by dmigliacci** [reply]
è andato, top!
